### ISSUE TITLE
Error: Cannot find module './Coder'

### ISSUE THREAD

**Issue by macjabeth** [liwangqian/LuaCoderAssist#2]
Hello. I just installed the extension and this error shows up whenever I open a lua file:

```
module.js:472
    throw err;
    ^

Error: Cannot find module './Coder'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/boonie/.vscode-insiders/extensions/liwangqian.luacoderassist-1.2.1/server/server.js:7:15)
    at Object.<anonymous> (/home/boonie/.vscode-insiders/extensions/liwangqian.luacoderassist-1.2.1/server/server.js:114:3)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
[Info  - 4:18:48 AM] Connection to server got closed.
```

I think the import just needs to be lowercased.

**Comment by liwangqian** [comment]
@macjabeth I have fixed the bug in VER 1.2.2, try again please.

It's all work well in my windows system, I did not test in other system.

**Comment by macjabeth** [comment]
@liwangqian Cheers! I'm not seeing the error anymore, so it's safe to say this issue has been fixed.
